### PR TITLE
Intl power lifting data

### PR DESCRIPTION
**Pull request by Swaha294**

### CODE REVIEW

File: R/datasets.R [hunk 1]
```diff
@@ -362,3 +362,8 @@
 #' }
 #' @seealso This data was sampled from the `movies` data frame in the `ggplot2movies` package.
 "movies_sample"
+
+
+#' International Power Lifting Results
```
Comment (mariumtapal) [inline R/datasets.R:367]: Please add the documentation. There are examples above!

File: data-raw/process_data_sets.R [hunk 2]
```diff
@@ -272,3 +272,11 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+## Adding international powerlifting data
+ipf_lifts <- readr::read_csv("https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2019/2019-10-08/ipf_lifts.csv") %>%
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: You don't need to call `readr::` since `tidyverse` is loaded at the beginning of this script.

Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Please pipe the URL/path to the `read_csv()`

File: data-raw/process_data_sets.R [hunk 3]
```diff
@@ -272,3 +272,11 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+## Adding international powerlifting data
+ipf_lifts <- readr::read_csv("https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2019/2019-10-08/ipf_lifts.csv") %>% 
+  janitor::clean_names()
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Again, you don't need to call `janitor::`

File: data-raw/process_data_sets.R (hunk 2 above)
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Also, if you are using the .csv stored on Github in another repository, what is the purpose of adding the .csv to `data-raw`?

File: data-raw/process_data_sets.R [hunk 4]
```diff
@@ -272,3 +272,11 @@ pennies_resamples <-
...
+
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Please delete this blank line for consistency!

File: data-raw/process_data_sets.R (hunk 2 above)
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Look at the formatting of the data cleaning in the above examples. The variable should be on one line and so on.

**Review by mariumtapal — CHANGES_REQUESTED** [review]

File: DESCRIPTION [hunk 5]
```diff
@@ -1,16 +1,19 @@
...
     person("Delaney", "Moran", email = "delaneyemoran@gmail.com", role = "ctb"),
     person("Evgeni", "Chasnovski", email = "evgeni.chasnovski@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-1617-4019")),
     person("Will", "Hopper", email = "wjhopper510@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-7848-1946")),
-    person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5093-6462"))
-    )
+    person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5093-6462")),
+    person("Swaha", "Bhattacharya", email = "itsswahabhattacharya@gmail.com", role = "ctb"),
+    person("Vivian", "Almaraz", email = "viviansofia101@gmail.com", role = "ctb"),
+    person("Elle Jo", "Whalen", email = "ewhalen@smith.edu", role = "ctb"),
+    person("Jacqueline", "Chen", email = "jchen76@smith.edu", role = "ctb"))
```
Comment (mariumtapal) [inline DESCRIPTION]: Please move the last parenthesis to a new line to avoid merge conflicts later.

File: NEWS.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
+# moderndive 0.5.3.9001
```
Comment (mariumtapal) [inline NEWS.md]: Change version number to 0.5.4 here

File: data-raw/process_data_sets.R [hunk 7]
```diff
@@ -272,3 +272,12 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+## Adding international powerlifting data
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R:278]: Delete the `R/iplf_lifts.R` file from the PR.

File: data-raw/process_data_sets.R [hunk 8]
```diff
@@ -272,3 +272,12 @@ pennies_resamples <-
...
+ipf_lifts <- 
+  "data-raw/IPF Lifts - ipf_lifts.csv" %>%
+  read_csv() %>%
+  clean_names()
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Look at the data types of the variables. `sex`, `equipment`, `event` all should be factors, not characters. This is not an exhaustive list of changes.

**Review by mariumtapal — CHANGES_REQUESTED** [review]

File: R/datasets.R [hunk 9]
```diff
@@ -362,3 +362,29 @@
...
+#'   \item{weight_class_kg}{The weight class in which the lifter competed, to two decimal places}
+#'   \item{best3squat_kg}{Maximum of the first three successful attempts for the lift}
+#'   \item{best3bench_kg}{Maximum of the first three successful attempts for the lift}
+#'   \item{best3deadlift_kg}{Maximum of the first three successful attempts for the lift}
+#'   \item{place}{The recorded place of the lifter in the given division at the end of the meet}
+#'   \item{date}{Date of the event}
+#'   \item{federation}{The federation that hosted the meet}
+#'   \item{meet_name}{The name of the meet}
+#' }
+#' @seealso This data is a subset of the open dataset [Open Powerlifting](https://www.openpowerlifting.org/)
```
Comment (beanumber) [inline R/datasets.R]: Can you please add a `@source` directive?

**Review by beanumber — CHANGES_REQUESTED** [review]
We're getting close here...

File: data-raw/process_data_sets.R [hunk 10]
```diff
@@ -272,3 +272,24 @@ pennies_resamples <-
...
+  clean_names() %>% 
+  mutate(
+    name = as.factor(name),
+    sex = as.factor(sex),
+    event = as.factor(event),
+    equipment = as.factor(equipment),
+    age_class = as.factor(age_class),
+    division = as.factor(division),
+    weight_class_kg = as.factor(weight_class_kg),
+    place = as.factor(place),
```
Comment (beanumber) [inline data-raw/process_data_sets.R]: `place` should not be a `factor`, it should be `character`.

File: data-raw/process_data_sets.R [hunk 11]
```diff
@@ -272,3 +272,24 @@ pennies_resamples <-
...
+
+
+
+## Adding international powerlifting data
+ipf_lifts <- 
+  "data-raw/IPF Lifts - ipf_lifts.csv" %>%
+  read_csv() %>%
+  clean_names() %>% 
+  mutate(
+    name = as.factor(name),
```
Comment (beanumber) [inline data-raw/process_data_sets.R]: Do we want `name` to be a `factor`? I think this should stay as `character`.

File: data-raw/process_data_sets.R (hunk 10 above)
Comment (beanumber) [reply data-raw/process_data_sets.R]: Or, if you're going to make it a factor, you should order the levels intelligently.

File: data-raw/process_data_sets.R [hunk 12]
```diff
@@ -272,3 +272,24 @@ pennies_resamples <-
...
+    sex = as.factor(sex),
+    event = as.factor(event),
+    equipment = as.factor(equipment),
+    age_class = as.factor(age_class),
+    division = as.factor(division),
+    weight_class_kg = as.factor(weight_class_kg),
+    place = as.factor(place),
+    federation = as.factor(federation),
+    meet_name = as.factor(meet_name)
```
Comment (beanumber) [inline data-raw/process_data_sets.R]: I don't think `meet_name` needs to be a factor.

File: data-raw/process_data_sets.R [hunk 13]
```diff
@@ -272,3 +272,24 @@ pennies_resamples <-
...
+  read_csv() %>%
+  clean_names() %>% 
+  mutate(
+    name = as.factor(name),
+    sex = as.factor(sex),
+    event = as.factor(event),
+    equipment = as.factor(equipment),
+    age_class = as.factor(age_class),
+    division = as.factor(division),
+    weight_class_kg = as.factor(weight_class_kg),
```
Comment (beanumber) [inline data-raw/process_data_sets.R]: If this is going to be a `factor`, you should order the levels intelligently.

File: DESCRIPTION [hunk 14]
```diff
@@ -1,15 +1,19 @@
 Package: moderndive
 Type: Package
 Title: Tidyverse-Friendly Introductory Linear Regression
-Version: 0.5.3.9000
+Version: 0.5.4
```
Comment (beanumber) [inline DESCRIPTION]: This should be 0.5.3.9001.

**Review by beanumber — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 6 above)
Comment (beanumber) [reply NEWS.md]: This still needs attention

**Review by beanumber — CHANGES_REQUESTED** [review]

**Comment by Swaha294** [comment]
@beanumber Professor, we're unsure of what else we need to add to the NEWS.md file. Could you please help us?

File: NEWS.md (hunk 6 above)
Comment (beanumber) [reply NEWS.md]: See @mariumtapal 's comment. You want the version number in the NEWS.md file to be 0.5.4.

**Review by beanumber — CHANGES_REQUESTED** [review]

**Review by beanumber — APPROVED** [review]
Good work!

**Pull request merged**